docs/docs/getting_started/00_nargo_installation.md

```diff
@@ -233,6 +218,14 @@ If you have hesitations with using direnv, you can launch a subshell with `nix d
 
 Advanced: If you aren't using direnv nor launching your editor within the subshell, you can try to install Barretenberg and other global dependencies the package needs. This is an advanced workflow and likely won't receive support!
 
+### Option 4: WSL (for Windows)
+
+Windows is not directly supported at this time. To install Noir on a Windows machine, it is recommended to use WSL.
```

Comment:
One thing to note incase we do get asked in the future; Its not Noir that doesn't support windows, its our default backend barretenberg which doesn't spit out windows binaries and cannot be compiled in windows (atleast no-one has tried recently and its not been tested)

Reply:
thanks, just clarified it a bit